f/radio/fn_radioChannels.sqf

```diff
@@ -0,0 +1,89 @@
+// F3 - Folk ARPS Long-Range Radio Module
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+
+/* ========================
+This module is activated in init.sqf.
+Example:
+[true] spawn f_fnc_radioChannels;
+
+Arguments:
+0. Long-range split mode (boolean). If true, each radio backpack type has a separate channel. If false, all backpacks use one channel. Default false.
+1. Number of channels to create (number, optional). If this is set, the number of channels to be created will be limited to this number. If channels are not split, this will be automatically set to 1.
+
+This module requires some missionmaker configuration. If you want to enable vehicle radio access, you need to specify vehicle classes or variable names in "f\radio\f_channelsList.sqf".
+=========================== */
+
+params [["_splitMode",false],["_channelCount",10]];
+
+// Disable the Command channel to avoid cheating
+2 enableChannel false;
```

Comment:
Do we also want to (ensure) other channels like global and side are disabled as well?

Reply:
I think it's best to leave those. The server already disables voice (at least for side) and the text chat is legitimately used for pre-mission banter and meta stuff. The main goal of this is a CC replacement so I don't want to intrude too far into removing commonly-used functionality outside of that.